### ISSUE TITLE
[EGP] Lua error: Object doesn't exist

### ISSUE THREAD

**Issue by KemGus** [wiremod/wire#2546]
How to reproduce:

I will provide you with an E2 that is causing error (see files below, keep in mind that you need to download both of them since one of them is lib)

1. press on the button with gmod icon on top left
2. Close it with the button on the right side of the window.
3. Get the error
4. Error:

```
[wire-master] addons/wire-master/lua/entities/gmod_wire_egp/lib/egplib/objectcontrol.lua:182: attempt to index local 'obj' (a nil value)
  1. PerformReorder - addons/wire-master/lua/entities/gmod_wire_egp/lib/egplib/objectcontrol.lua:182
   2. Receive - addons/wire-master/lua/entities/gmod_wire_egp/lib/egplib/transmitreceive.lua:551
    3. func - addons/wire-master/lua/entities/gmod_wire_egp/lib/egplib/transmitreceive.lua:557
     4. unknown - lua/includes/extensions/net.lua:33

sv: Expression 2 (TurboOS  Version 0.0.1
Made By KemGus
): Error(s) occured while executing 'postexecute' hook:
addons/wire-master/lua/entities/gmod_wire_egp/lib/egplib/objectcontrol.lua:182: attempt to index local 'obj' (a nil value)
```
[egp_functions.txt](https://github.com/wiremod/wire/files/10889073/egp_functions.txt)
[os_.txt](https://github.com/wiremod/wire/files/10889074/os_.txt)

Notes:
For some reason, when I close the window by clicking on the icon again, it doesn't error that often.
Sometimes error won't kick in, so the issue seems a bit random.

**Comment by KemGus** [comment]
Removing this part of the code (egp order one) fixed the issue
![image](https://user-images.githubusercontent.com/42640259/223578710-6239719b-c176-4084-a0fc-45918f4c73e5.png)

**Comment by Denneisk** [comment]
I believe this is being caused because it's trying to reorder on an object that is deleted but the rendertable doesn't update in time. A simple null check seems to be a stopgap solution.

**Comment by Denneisk** [comment]
Can you confirm if this still happens? Might have been coincidentally fixed in #2731.

**Comment by Denneisk** [comment]
Can't reproduce so I'm going to assume this has been fixed.
